.gitlab/deploy_packages/oci.yml

```
@@ -38,3 +38,8 @@ deploy_installer_oci:
   needs: [ "installer_oci", "go_tools_deps" ]
   variables:
     OCI_PRODUCT: "datadog-installer"
+  script:
+    - *deploy_packages_script
```

```suggestion
    - [.deploy_packages_oci, script]
```
Should be equivalent. We'd rather not add yet another way to reuse existing code within yaml/